docs/topics/modules/getting-osio-authentication-token.adoc

```diff
@@ -0,0 +1,21 @@
+[id="getting_osio_authentication"]
+= Getting OpenShift.io authentication token
+
+You require an {osio} authentication token to invoke all Fabric8-Analytics API resources.
+
+Get the {osio} authentication token as follows:
+
+. Log into link:https://openshift.io[{osio}]
+. Open the user profile.
++
+image::profile.png[user profile]
```

Comment:
We don't need to do the red circles outside of the GSG. And we'll be getting rid of them even there. In fact, we don't want them if at all possible.

Reply:
Also, images should be named better -- to make it possible to recognize where they belong anmd what's on them from the name itself. `profile.png` is very vague.

@mishaone FYI, too.